frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/createNamespace.ts

```diff
@@ -0,0 +1,55 @@
+import { checkErrors, testName } from '../../../integration-tests-cypress/support';
+import { listPage } from '../../../integration-tests-cypress/views/list-page';
+import { modal } from '../../../integration-tests-cypress/views/modal';
+
+describe('Create namespace from install operators', () => {
+  before(() => {
+    cy.login();
+    cy.createProject(testName);
+  });
+
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  const nsName = `${testName}-ns`;
+
+  it('creates namespace from operator install page', () => {
+    const operatorSelector = '3scale-community-operator-community-operators-openshift-marketplace';
+    cy.log('test namespace creation from dropdown');
+    cy.visit(`/operatorhub/ns/${testName}`);
+    cy.byTestID(operatorSelector).click();
+    cy.byLegacyTestID('operator-install-btn').click({ force: true });
```

Comment:
I had to force the click (as a workaround) since cypress wasn't able to identify the `Install` button from the pop-up that opens after selecting an operator on the install operators page. I tried different selectors but had no luck. Not sure why cypress isn't able to recognize this (maybe the focus shifts or that it is not a button element?). Please suggest if there is any way of achieving this.

Reply:
What message does Cypress give when this happens?

cc @dtaylor113

Reply:
Cypress waits for 7-10 seconds to locate the UI element and says the element is not visible. However, when I cycle through the snapshot previews of cypress, I see that it highlights the button, but the `cy.get()` just doesn't work.
![install_button](https://user-images.githubusercontent.com/39300150/94964239-7d2c7280-04c7-11eb-8af5-a0a278341567.png)

Reply:
I'm wondering if this is caused by the `#modal-container` element, which has a calculated height of 0. I was reading over https://github.com/cypress-io/cypress/issues/695 and decided to inspect that button and it's parent elements to make sure none of them had an effective height/width of zero, and found this.

Reply:
![Screen Shot 2020-10-05 at 11 27 23 AM](https://user-images.githubusercontent.com/22625502/95101114-ce26ab80-06ff-11eb-956a-fb70cac18199.png)